src/libs/workspace-utils.ts

```diff
@@ -1,16 +1,16 @@
 
-export type CloudProviderType = 'AZURE' | 'GCP'
-export const cloudProviderTypes: Record<CloudProviderType, CloudProviderType> = {
+export type CloudProvider = 'AZURE' | 'GCP'
```

Comment:
This should be CloudPlatform now

Reply:
Yeah, I was a bit hesitant to change these since they already exist and you had a ticket to make the names consistent... I would prefer a separate PR to rename these since the surface of this PR is already large